### PR TITLE
❏ div html

### PR DESCRIPTION
**Pull request by Melkortheslumpgod**
please include my sicko html codes. sir 🤠

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -31,7 +31,18 @@ <h1>Welcome to the student showcase!</h1>
         <div>ELEVEN</div>
         <div>TWELVE</div>
         <div>THIRTEEN</div>
-        <div>FOURTEEN</div>
+        <div>JOE ASARO
+            <hr>
+            <img src="https://i.imgur.com/ZnCvaDN.jpg" alt="cute italian boy with trash can" width="42"><br>
+           <em> If you want to teach people a new way of thinking, don’t bother trying to teach them. Instead, give them a tool, the use of which will lead to new ways of thinking.
+<br>– Richard Buckminster Fuller</em>
```
Comment (NAlexPear) [inline index.html]: You can do better than `<br/>`! Perhaps you want a `<p>`?

File: index.html [hunk 2]
```diff
@@ -31,7 +31,18 @@ <h1>Welcome to the student showcase!</h1>
         <div>ELEVEN</div>
         <div>TWELVE</div>
         <div>THIRTEEN</div>
-        <div>FOURTEEN</div>
+        <div>JOE ASARO
+            <hr>
+            <img src="https://i.imgur.com/ZnCvaDN.jpg" alt="cute italian boy with trash can" width="42"><br>
+           <em> If you want to teach people a new way of thinking, don’t bother trying to teach them. Instead, give them a tool, the use of which will lead to new ways of thinking.
```
Comment (NAlexPear) [inline index.html]: `font-style: italic` > `em`

File: index.html [hunk 3]
```diff
@@ -31,7 +31,18 @@ <h1>Welcome to the student showcase!</h1>
         <div>ELEVEN</div>
         <div>TWELVE</div>
         <div>THIRTEEN</div>
-        <div>FOURTEEN</div>
+        <div>JOE ASARO
+            <hr>
+            <img src="https://i.imgur.com/ZnCvaDN.jpg" alt="cute italian boy with trash can" width="42"><br>
```
Comment (NAlexPear) [inline index.html]: 💩 _inline styles_ 💩

File: index.html [hunk 4]
```diff
@@ -31,7 +31,18 @@ <h1>Welcome to the student showcase!</h1>
         <div>ELEVEN</div>
         <div>TWELVE</div>
         <div>THIRTEEN</div>
-        <div>FOURTEEN</div>
+        <div>JOE ASARO
```
Comment (NAlexPear) [inline index.html]: make sure that there's a new line between this `<div>` and `JOE ASARO`

File: index.html [hunk 5]
```diff
@@ -31,7 +31,18 @@ <h1>Welcome to the student showcase!</h1>
...
+            <hr>
+            <img src="https://i.imgur.com/ZnCvaDN.jpg" alt="cute italian boy with trash can" width="42"><br>
+           <em> If you want to teach people a new way of thinking, don’t bother trying to teach them. Instead, give them a tool, the use of which will lead to new ways of thinking.
+<br>– Richard Buckminster Fuller</em>
+<ul>
+    <li>J ASARO</li>
+    <li>St. Louis, MO</li>
+    <li>(314)599-5936 </li>
+    <li><a href="joeyasaro@gmail.com">joeyasaro@gmail.com</a></li>
+</ul>
```
Comment (NAlexPear) [inline index.html]: Make sure that this `<ul>` is indented correctly!

File: index.html [hunk 6]
```diff
@@ -31,7 +31,18 @@ <h1>Welcome to the student showcase!</h1>
         <div>ELEVEN</div>
         <div>TWELVE</div>
         <div>THIRTEEN</div>
-        <div>FOURTEEN</div>
+        <div>
+            <h1>JOE ASARO</h1>
+            <img src="https://i.imgur.com/ZnCvaDN.jpg" alt="cute italian boy with trash can" width="42"><p>
```
Comment (NAlexPear) [inline index.html]: Make sure that there's a line break between `img` and `p`, here

File: index.html [hunk 7]
```diff
@@ -31,7 +31,18 @@ <h1>Welcome to the student showcase!</h1>
...
+            If you want to teach people a new way of thinking, don’t bother trying to teach them. Instead, give them a tool, the use of which will lead to new ways of thinking.
+            – Richard Buckminster Fuller</p>
+                <ul>
```
Comment (NAlexPear) [inline index.html]: Check the indentation of this unordered list!

Comment (Melkortheslumpgod) [reply index.html]: what is the correct indentation?

**Pull request merged**